nomad/server.go

```diff
@@ -983,3 +1054,30 @@ func (s *Server) Datacenter() string {
 func (s *Server) GetConfig() *Config {
 	return s.config
 }
+
+// TODO(alex) we need a outage guide
+// peersInfoContent is used to help operators understand what happened to the
+// peers.json file. This is written to a file called peers.info in the same
+// location.
+const peersInfoContent = `
+As of Nomad 0.5.6, the peers.json file is only used for recovery
+after an outage. It should be formatted as a JSON array containing the address
+and port of each Consul server in the cluster, like this:
+
+["10.1.0.1:8500","10.1.0.2:8500","10.1.0.3:8500"]
```

Comment:
These were wrong in the initial PR for Consul, and don't look right for Nomad. Should update these port numbers.